### DISCUSSION TITLE
problem executing the IMAGE_TO_RAM=1 option

### DISCUSSION THREAD

**Discussion by oliviernet62** [ltsp/ltsp#750] [Q&A]
We have been maintaining the ltsp system for 15 years on our computer park of around 150 workstations.
We have migrated our fleet from ltsp5 to ltsp6.
I have a question concerning the execution of the IMAGE_TO_RAM=1 option. I wrote this option in the ltsp.conf file and How is it possible to verify that this option is indeed active and executed on the ltsp client.
I notice that the launch of applications remains quite slow during the first launch.
In the ltsp file the option DEFAULT_IMAGE= and MENU_TIMEOUT does not work either
I am attaching our ltsp file and mounting resources

Thank you for your help
[ltsp.txt](https://github.com/ltsp/ltsp/files/9586467/ltsp.txt)
[mount.txt](https://github.com/ltsp/ltsp/files/9586468/mount.txt)

**Comment by alkisg** [comment]
Regarding IMAGE_TO_RAM I cannot comment, I didn't author it and I'm not using it.

Regarding DEFAULT_IMAGE and MENU_TIMEOUT, after setting these in ltsp.conf, you need to run `sudo ltsp ipxe`.
